microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1462.37
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1418.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)

```

Comment:
```suggestion
[NuGet package for WebView2 SDK 1.0.1462.37](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)
```

Reply:
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/?view=webview2-1.0.1462.37

Reply:
fixed local version

Reply:
current version is right